app/controllers/orders/index.js

```diff
@@ -4,5 +4,14 @@ export default searchModule.extend({
 
   searchModelName: "designation",
   unloadAll: true,
-  minSearchTextLength: 2
+  minSearchTextLength: 2,
+
+  onItemLoaded(record) {
+    const orgId = record.get("gcOrganisationId");
+    if (orgId) {
+      this.store.findRecord("gc_organisation", orgId, { reload: false });
```

Comment:
@patrixr , is there any reason for and extra request for loading `gc_organisation` related to Order??
Order response doesn't contain  `gc_organisation`?